src/ui/theme/recipes/LinkRecipe.ts

```diff
@@ -39,10 +39,13 @@ export const linkRecipe = defineRecipe({
         },
       },
       buttonLink: {
-        textDecoration: 'none',
+        p: '0 !important',
+        pb: '0.5 !important',
+        textDecoration: 'none !important',
```

Comment:
We need to understand why !important is necessary here, it can cause many issue in future.

Reply:
Important is required because the styles from the default variant are applied and take precedence. IMO this is a bug on Chakra UI's as the defaultVariant should just be the default variant that is applied in the absence of any other variant being specified. If we want to fix this issue ourselves by removing the use of the default variant, I think that should be done in another PR, because it will require changing numerous files